extensions/wikia/Email/templates/src/emailPreview.mustache

```diff
@@ -1,4 +1,6 @@
-{{#body}}{{{body}}}{{/body}}{{^body}}
+{{{body}}}
+
```

Comment:
There's no user-facing difference, and the empty line makes this code more clear.